Doc/library/calendar.rst

```diff
@@ -469,6 +469,29 @@ The :mod:`calendar` module exports the following data attributes:
 
    Aliases for day numbers, where ``MONDAY`` is ``0`` and ``SUNDAY`` is ``6``.
 
+
+The :mod:`calendar` module defines the following exceptions:
+
+.. exception:: IllegalMonthError(month)
+
+   A subclass of :exc:`ValueError`,
+   raised when the given month number is outwith the range 1-12 (inclusive).
```

Comment:
I had to look this up, let's use something more widely understood than a Scottish English word:

```suggestion
   raised when the given month number is outside the range 1-12 (inclusive).
```

Reply:
Oh, fair enough. Though it's Scottish, rather than English! A better translation is closer to "outside of", will push a change.